migrations/Version20200101000000.php

```diff
@@ -0,0 +1,824 @@
+<?php
```

Comment:
so how is this used ? the server manager will create the database , and then run the migrations on it ? And in that way it creates the tables ?